### ISSUE TITLE
Installing Node 10 doesn't work: "npm is known not to run on Node.js v10.18.1"

### ISSUE THREAD

**Issue by Jabark** [Schniz/fnm#1101]
When I install Node 10, in particular 10.18.1, it tells me:
> npm is known not to run on Node.js v10.18.1
> You'll need to upgrade to a newer Node.js version in order to use this
> version of npm. You can find the latest version at http://nodejs.org

The interesting part is that I was using 10.18.1 perfectly fine before switching to use `fnm`, as are all my colleagues. I double checked, and the NPM version that is downloaded is the same _6.13.4_ that my colleagues and my "non-fnm" version of 10.18.1 uses.

![image](https://github.com/Schniz/fnm/assets/9607853/21c67a55-587e-4bf6-a7e7-68241f4c97dc)

Any suggestions please?

Things that I have tried and failed:
- uninstalling and reinstalling.
- copying and pasting across npm from my "non-fnm" version.

**Comment by Schniz** [comment]
That's an interesting issue but I'm not going to invest time on this as v10 is EOL for years now

**Comment by Jabark** [comment]
That's a shame, considering the whole point of a tool like this is to be able to flip between versions as and when needed. Guess we can't use it then 🤷🏻‍♂️.

**Comment by Jabark** [comment]
Hi @Schniz, I found that the issue was because it was trying to use the npm version for the 20.8.0 version of node that I had installed before moving back to 10.18.1. Is there a way to make the npm version automatically change when the node version is changed using fnm? Currently I'm having to manually flip between npm versions 6.13.4 and 10.8.1 every time I flip between node 10 and 20. I have each version of npm installed into the node_modules directory within each version of node, it just doesn't seem to use it...
